extensions/interactions/CodeRepl/CodeRepl.html

```diff
@@ -17,6 +17,10 @@
     color: red;
     font-weight: bold;
   }
+    .code-repl-marker{
```

Comment:
This should follow the existing indentation (two spaces).

Also, I looked into this a bit more deeply and I think the .code-repl-marker is good for making the code readonly, but isn't great for setting the CSS class. I would suggest removing the .code-repl-marker class, and, in addition to markText, also adding a call to

     editor.addLineClass(rowIndex, "text", className);

You can use the following styling:

```
    .CodeMirror-code .code-repl-noneditable-line {
      background-color: #eee;
      opacity: 0.8;
      width: 100%;
    }
```